pkg/controller/migmigration/backup.go

```diff
@@ -305,17 +382,23 @@ func (t *Task) annotateStorageResources() error {
 		client.Update(context.TODO(), &resource)
 	}
 
+	resticPodCount := 0
```

Comment:
What does `resticPodCount` represent? Is this the number of pods in the restic daemonset? e.g. == the number of nodes in the cluster?

Reply:
Changing this to `resticAnnotationCount`. This is the number of pods that have a restic annotation